Clarify when local_app_support() is needed vs package functions (#438)
rstudio/shinytest2: vignettes/zzz-faq.Rmd

This is not automatically performed as it may be costly to load the support environment.

> **Important note**: If your app is defined inline (as a Shiny app object passed directly to `AppDriver$new()`), you typically do **NOT** need these functions. Your package functions are already available via `library(yourpackage)` or `devtools::load_all()`.

Reviewer: ```suggestion
**Important note**: If your app is defined inline (as a Shiny app object passed directly to `AppDriver$new()`), you typically do **NOT** need these functions. Your package functions are already available when `{testthat}` loads your local package.
```
